pilot/pkg/networking/v1alpha3/listener.go

```diff
@@ -17,11 +17,23 @@ package v1alpha3
 import (
 	xdsapi "github.com/envoyproxy/go-control-plane/envoy/api/v2"
 
+	"github.com/gogo/protobuf/types"
 	_ "github.com/golang/glog" // nolint
 
+	"fmt"
+	"strings"
+
+	"github.com/envoyproxy/go-control-plane/envoy/api/v2/listener"
+
 	"istio.io/istio/pilot/pkg/model"
 )
 
+const (
+	// istioIngress is the name of the service running the Istio Ingress controller
+	istioIngress               = "istio-ingress"
+	envoyHTTPConnectionManager = "envoy.http_connection_manager"
```

Comment:
put this constant together with the others starting with envoy.*

Reply:
Done - I want to keep the constants in the file using them if it's single-use.